ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -0,0 +1,668 @@
+import {Button} from 'app/components/buttons';
+import {FormSection} from 'app/components/forms';
+import {Header} from 'app/components/headers';
+
+import {
+  InfoIcon,
+  ValidationIcon
+} from 'app/components/icons';
+
+import {
+  Error,
+  ErrorMessage,
+  RadioButton,
+  styles as inputStyles,
+  TextArea,
+  TextInput,
+  ValidationError
+} from 'app/components/inputs';
+
+import {
+  TooltipTrigger
+} from 'app/components/popups';
+
+import {
+  profileApi
+} from 'app/services/swagger-fetch-clients';
+
+import {
+  DataAccessLevel,
+  Profile,
+} from 'generated/fetch/api';
+
+import colors from 'app/styles/colors';
+import {summarizeErrors} from 'app/utils/index';
+import {environment} from 'environments/environment';
+import * as fp from 'lodash/fp';
+import {Dropdown} from 'primereact/dropdown';
+import * as React from 'react';
+import * as validate from 'validate.js';
+
+function isBlank(s: string) {
```

Comment:
is there a builtin or library function for this? If we need to roll our own, then we need to have unit tests I think.